src/helpers/priceString.js

```diff
@@ -0,0 +1,5 @@
+function priceString(tv) {
+    return `\u20AC${tv.price},-`;
```

Comment:
Ik zie dat je de Unicode representatie van het euro-teken hebt gebruikt. Geen idee wat beter is, dit of direct het euroteken toevoegen aan de string.